src/theory/arith/arith_rewriter.cpp

```diff
@@ -83,9 +84,12 @@ RewriteResponse ArithRewriter::rewriteMinus(TNode t, bool pre){
 RewriteResponse ArithRewriter::rewriteUMinus(TNode t, bool pre){
   Assert(t.getKind() == kind::UMINUS);
 
-  if(t[0].getKind() == kind::CONST_RATIONAL){
+  if (t[0].isConst())
```

Comment:
Technically this removes the check **which** type of constant this is. I assume the assertion above implicitly assures it is arithmetic?

Reply:
Yes, the type rule for these kinds will ensure this is the case.